tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -5328,6 +5309,32 @@ Result D3D12Device::initialize(const Desc& desc)
         profileName,
         makeArray(slang::PreprocessorMacroDesc{"__D3D12__", "1"}).getView()));
 
+    {
+        D3D12_INDIRECT_ARGUMENT_DESC args[1];
+        args[0].Type = D3D12_INDIRECT_ARGUMENT_TYPE_DRAW;
+
+        D3D12_COMMAND_SIGNATURE_DESC desc;
+        desc.ByteStride = 36;
+        desc.NumArgumentDescs = 1;
+        desc.pArgumentDescs = args;
+        desc.NodeMask = 0;
+
+        SLANG_RETURN_ON_FAIL(m_device->CreateCommandSignature(&desc, nullptr, IID_PPV_ARGS(drawIndirectCmdSignature.writeRef())));
```

Comment:
Comment this block, maybe, to describe what it is doing? I think the short version is something like:

```
// Allocate a D3D12 "command signature" object that matches the behavior
// of a D3D11-style `DrawInstancedIndirect` operation.
```